### PR TITLE
Ci add cachix

### PR DESCRIPTION
**Pull request by Profpatsch**
Drawn out of https://github.com/target/lorri/pull/61

Adds support for Cachix to CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 "git":
   "depth": !!bool >-
     false
-"languge": >-
+"language": >-
```
Comment (grahamc) [inline .travis.yml]: :joy:

File: .travis.yml.nix [hunk 2]
```diff
@@ -20,34 +22,44 @@ let
...
+            cat $(nix-build --quiet ./.travis.yml.nix --no-out-link) > .travis.yml
+          lorri_travis_fold travis-yml-idempotent \
+            git diff -q ./.travis.yml
+          lorri_travis_fold carnix-idempotent \
+            git diff -q ./Cargo.nix
+        ''
+        # push test suite closure to cachix
+        ''
+          buildInputs=$(nix-build -E '(import ./shell.nix { isDevelopmentShell = false; }).buildInputs')
+          printf '%s' "$buildInputs" >> ${cachix-queue-file}
```
Comment (grahamc) [inline .travis.yml.nix]: why `printf` instead of just ` >> $HOME/push-to-cachix`?

Comment (Profpatsch) [reply .travis.yml.nix]: Ah, that’s an artifact from backporting from a later commit, will change.

**Comment by Profpatsch** [comment]
> I'm seeing that shellcheck is failing, but the build result isn't representing that:

Didn’t we fix this in https://github.com/target/lorri/pull/155? I wonder why it doesn’t fail CI.

**Comment by grahamc** [comment]
Thanks!

**Pull request merged**